src/components/LegacyModal/Modal.tsx

```diff
@@ -26,7 +26,7 @@ const Fade = ({ ...props }: any) => (
   <CSSTransition {...props} timeout={300} classNames="animate-modal" />
 );
 
-export class Modal extends PureComponent<Props, {}> {
+export class Modal extends PureComponent<Props, TObject> {
```

Comment:
If the component doesn't have state, I'd just use `null` here.

```suggestion
export class Modal extends PureComponent<Props, null> {
```

Reply:
Done